src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1810,7 +1810,21 @@ public function insert_posts( $items = [] ) {
 							}
 
 							if ( empty( $venue_unique_field ) || ( $lookup_venues_by_title && empty( $venue ) ) ) {
-								$venue = get_page_by_title( $event['Venue']['Venue'], 'OBJECT', Tribe__Events__Venue::POSTTYPE );
+								$venue_query = new WP_Query(
+									[
+										'post_type'      => Tribe__Events__Venue::POSTTYPE,
+										'title'          => $event['Venue']['Venue'],
+										'post_status'    => 'all',
+										'posts_per_page' => 1,
+										'no_found_rows'  => true,
+										'ignore_sticky_posts' => true,
+										'update_post_term_cache' => false,
+										'orderby'        => 'post_date ID',
```

Comment:
Align the `=>`

Reply:
About that... I had them all aligned and `phpcs` threw a fit! This is the spacing it accepted, oddly enough. Have we been having issues with that?

Reply:
See here: https://github.com/the-events-calendar/the-events-calendar/actions/runs/12755046123/job/35550252359